server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/MessageAppender.java

```diff
@@ -141,9 +141,9 @@ public MessageAppender(MailboxManager mailboxManager, MessageIdManager messageId
         }
     }
 
-    public MessageFactory.MetaDataWithContent appendMessageInMailbox(CreationMessageEntry createdEntry,
-                                                                       MailboxId targetMailbox,
-                                                                       MailboxSession session) throws MailboxException {
+    public MessageFullViewFactory.MetaDataWithContent appendMessageInMailbox(CreationMessageEntry createdEntry,
```

Comment:
Can we import MessageFullViewFactory.MetaDataWithContent so that we don't have the MessageFullViewFactory prefix?